driverless/src/main/java/com/upgrad/hirewheels/HireWheelsApplication.java

```diff
@@ -1,6 +1,5 @@
 package com.upgrad.hirewheels;
```

Comment:
As the location table has to be hardcoded in the frontend we must have deterministic values of the Ids for each location in the location table. As of now, it's auto generated. We should change this otherwise location Ids corresponding to each location changes every time the application is restarted.